src/service/processing_matches.py

```diff
@@ -81,6 +83,19 @@ async def create_match(
     return curr_match
 
 
+def _process_user_params(params: dict[str, Any], matcher: Matcher):
+    ret = jsonschema_default.create_from(matcher.user_parameters)
+    if params:
+        ret = mergedeep.merge(ret, params, strategy=mergedeep.Strategy.REPLACE)
```

Comment:
Just wanna clarify here: 
When `ret` and `params` keys are the same, we take the value from `params`?

Reply:
That does not sound right ...

Reply:
Right. 2nd argument keys replace 1st

Reply:
that does not sound right ...

Reply:
Why not?